avogadro/qtplugins/select/select.cpp

```diff
@@ -438,11 +438,13 @@ void Select::selectAtomIndex()
     // check if it's a range
     if (item.contains('-')) {
       auto range = item.split('-');
-      if (range.size() >= 2) {
```

Comment:
But what about the case with "1-2,5-6,8,10-12" ?

Reply:
it will work sir i changed the range.size()==2 
because of this case 8-9-7 which is valid in that if loop but doesnt make sense

Reply:
it will crash the system if we dont remove the case of 5-6-7

Reply:
@ghutchis  sir can u look in this pr?